core/src/main/java/io/crate/Version.java

```diff
@@ -41,7 +41,7 @@
 
 
     public static final boolean SNAPSHOT = false;
-    public static final Version CURRENT = new Version(2020599, SNAPSHOT, org.elasticsearch.Version.V_5_5_2_UNRELEASED);
+    public static final Version CURRENT = new Version(2020699, SNAPSHOT, org.elasticsearch.Version.V_5_5_2_UNRELEASED);
```

Comment:
How is this version number generated? (`2020699`)

Reply:
The ID numbers are formatted as ABBCCDD where:
A is the major version (here 2)
BB is the minor version (here 02)
and CC is the revision (here being bumped from 05 to 06)
DD is the beta or release candidate indicator. < 50 are Beta builds, < 99 are Release Candidates and 99 is a release. We typically don't change this number

Reply:
Ahh I see, thanks for explaining!